### PR TITLE
Add Yolo V3 Tiny from Keras

### PR DESCRIPTION
**Pull request by eizamaliev**
TODO

- [x] Add path to weight and JSON
- [x] Add GFLOPs and mParams values in documentation and index.md 

cc @yssemaev

### CODE REVIEW

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+YOLO v3 Tiny is a real-time object detection model implemented with Keras\* from this [repository](https://github.com/david8862/keras-YOLOv3-model-set) and converted to TensorFlow\* framework. This model was pretrained on COCO\* dataset with 80 classes.
+
+## Conversion
+
+## Conversion
+
+1. Download or clone the official [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `d38c3d8` commit).
+2. Use the folllowing commands to get original model (named `yolov3_tiny` in repository) and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/blob/ffede5d316568479610b75a3424e2a7b81f0209b/README.md)  file in the official repository):
+
+   1. Download YOLO v3 weights:
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:14]:
```suggestion
   1. Download YOLO v3 Tiny weights:
```
```suggestion
   1. Download YOLO v3 weights:
```

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        ```
+        wget -O weights/yolov3-tiny.weights https://pjreddie.com/media/files/yolov3-tiny.weights
+        ```
+
+   1. Convert model weights to Keras\*:
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:19]: it is ok on github as I know, but lead to incorrect rendering in OV docs
```suggestion
   2. Convert model weights to Keras\*:
```
```suggestion
   1. Convert model weights to Keras\*:
```

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+- `N` - number of detection boxes for cell
+- `Cx`, `Cy` - cell index
+
+Detection box has format [`x`,`y`,`h`,`w`,`box_score`,`class_no_1`, ..., `class_no_80`], where:
+- (`x`,`y`) - coordinates of box center relative to the cell
+- `h`,`w` - raw height and width of box, apply [exponential function](https://en.wikipedia.org/wiki/Exponential_function) and multiply by corresponding anchors to get absolute height and width values
+- `box_score` - confidence of detection box in [0,1] range
+- `class_no_1`,...,`class_no_80` - probability distribution over the classes in the [0,1] range, multiply by confidence value to get confidence of each class
+
+> **NOTE:** Original model uses [[3,4,5],[**1,2,3**]] (not [[3,4,5], [**0,1,2**]]) mask for anchors.
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:108]: not sure that it is right place for this comment, for me it will be better to specify anchor mask for each layer without emphasis that original repo has mistake

File: models/public/yolo-v3-tiny-tf/model.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  YOLO v3 Tiny is a real-time object detection model implemented with Keras* from
+  this repository <https://github.com/david8862/keras-YOLOv3-model-set> and converted
+  to TensorFlow* framework. This model was pretrained on COCO* dataset with 80 classes.
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/model.yml:18]: is it normal that in description \*, while her just *?

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+        wget -O weights/yolov3-tiny.weights https://pjreddie.com/media/files/yolov3-tiny.weights
+        ```
+
+   1. Convert model weights to Keras\*:
+        ```
+        python tools/convert.py cfg/yolov3.cfg weights/yolov3-tiny.weights weights/yolov3-tiny.h5
+        ```
+3. Convert model to protobuf:
+    ```
+    python tools/keras_to_tensorflow.py --input_model weights/yolov3-tiny.h5 --output_model="path/to/save/model.pb"
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:25]: generally, for variable paths used <> pattern

File: models/public/yolo-v3-tiny-tf/accuracy-check.yml [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+        metrics:
+          - type: map
+            integral: 11point
+            ignore_difficult: true
+            presenter: print_scalar
+          - type: coco_precision
+            max_detections: 100
+            threshold: 0.5
+
+global_definitions: ../dataset_definitions.yml
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/accuracy-check.yml:99]: path to global definitions is invalid, it is not located in models/public directory. Please remove this line

File: models/public/yolo-v3-tiny-tf/accuracy-check.yml [hunk 7]
```diff
@@ -0,0 +1,99 @@
+models:
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/accuracy-check.yml:1]: please add soft link on config to tools/accuracy_checker/configs with <model_name>.yml

File: models/public/yolo-v3-tiny-tf/model.yml (hunk 4 above)
Comment (eizamaliev) [reply models/public/yolo-v3-tiny-tf/model.yml:18]: Yes, because in markdown we use `\*` instead of `*` to escape it.

Comment (eaidova) [reply models/public/yolo-v3-tiny-tf/model.yml:18]: ok

File: models/public/yolo-v3-tiny-tf/accuracy-check.yml (hunk 7 above)
Comment (eizamaliev) [reply models/public/yolo-v3-tiny-tf/accuracy-check.yml:1]: Done

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+## Use Case and High-Level Description
+
+YOLO v3 Tiny is a real-time object detection model implemented with Keras\* from this [repository](https://github.com/david8862/keras-YOLOv3-model-set) and converted to TensorFlow\* framework. This model was pretrained on COCO\* dataset with 80 classes.
+
+## Conversion
+
+## Conversion
+
+1. Download or clone the official [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `d38c3d8` commit).
+2. Use the folllowing commands to get original model (named `yolov3_tiny` in repository) and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/blob/ffede5d316568479610b75a3424e2a7b81f0209b/README.md)  file in the official repository):
```
Comment (IRDonch) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:12]: Why is commit for the README different from the commit in item 1?

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+   1. Download YOLO v3 Tiny weights:
+        ```
+        wget -O weights/yolov3-tiny.weights https://pjreddie.com/media/files/yolov3-tiny.weights
+        ```
+
+   2. Convert model weights to Keras\*:
+        ```
+        python tools/convert.py cfg/yolov3.cfg weights/yolov3-tiny.weights weights/yolov3-tiny.h5
```
Comment (IRDonch) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:21]: Something is wrong here. There's no `tools/convert.py` script in the linked repository.

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+
+## Output
+
+### Original model
+
+1. The array of detection summary info, name - `conv2d_9/BiasAdd`,  shape - `1,13,13,255`. The anchor values are `81,82, 135,169, 344,319`.
+
+2. The array of detection summary info, name - `conv2d_12/BiasAdd`,  shape - `1,26,26,255`. The anchor values are `23,27, 37,58, 81,82`.
+
+For each case format is `B,Cx,Cy,N*85,`, where
```
Comment (IRDonch) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:80]:
```suggestion
For each case format is `B,Cx,Cy,N*85`, where
```

**Review by IRDonch — CHANGES_REQUESTED** [review]

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md (hunk 9 above)
Comment (eizamaliev) [reply models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md]: Fixed.

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+
+## Use Case and High-Level Description
+
+YOLO v3 Tiny is a real-time object detection model implemented with Keras\* from this [repository](https://github.com/david8862/keras-YOLOv3-model-set) and converted to TensorFlow\* framework. This model was pretrained on COCO\* dataset with 80 classes.
+
+## Conversion
+
+## Conversion
+
+1. Download or clone the official [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `d38c3d8` commit).
```
Comment (IRDonch) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:11]: I don't see how this repository can be considered "official" in any way. I think it would be better to use "original".

**Comment by IRDonch** [comment]
The model failed to get converted. Is this a known bug in MO from the last release?

**Comment by eaidova** [comment]
> The model failed to get converted. Is this a known bug in MO from the last release?

@eizamaliev @IRDonch looks like wrong input layer name provided

File: models/public/yolo-v3-tiny-tf/accuracy-check.yml [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+      - framework: tf
+        model:   yolo-v3-tiny.pb
+        adapter:
+          type: yolo_v3
+          anchors: tiny_yolo_v3
+          num: 6
+          coords: 4
+          classes: 80
+          threshold: 0.001
+          anchor_masks: [[3,4,5],[1,2,3]]
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/accuracy-check.yml:13]:
```suggestion
          anchor_masks: [[3, 4, 5], [1, 2, 3]]
```

File: models/public/yolo-v3-tiny-tf/accuracy-check.yml [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+          raw_output: True
+          output_format: HWB
+          cells: [13,26]
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/accuracy-check.yml:16]:
```suggestion
          cells: [13, 26]
```

File: models/public/yolo-v3-tiny-tf/accuracy-check.yml [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+          outputs:
+            - conv2d_9/BiasAdd
+            - conv2d_12/BiasAdd
+        inputs:
+          - name: 'image_input'
+            type: INPUT
+            shape: [1,416,416,3]
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/accuracy-check.yml:23]:
```suggestion
            shape: [1, 416, 416, 3]
```

File: models/public/yolo-v3-tiny-tf/accuracy-check.yml [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: yolo_v3
+          anchors: tiny_yolo_v3
+          num: 3
+          coords: 4
+          classes: 80
+          threshold: 0.001
+          anchor_masks: [[3,4,5],[1,2,3]]
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/accuracy-check.yml:69]:
```suggestion
          anchor_masks: [[3, 4, 5],[1 ,2, 3]]
```

File: models/public/yolo-v3-tiny-tf/accuracy-check.yml (hunk 14 above)
Comment (eaidova) [reply models/public/yolo-v3-tiny-tf/accuracy-check.yml:23]: BTW, if I right remember, current tf launcher implementation does not support shape specification. Do you update it?

File: models/public/yolo-v3-tiny-tf/model.yml [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+postprocessing:
+  - $type: unpack_archive
+    format: zip
+    file: yolo-v3-tiny-tf.zip
+model_optimizer_args:
+  - --input_shape=[1,416,416,3]
+  - --input=input_image
+  - --scale_values=input_image[255]
+  - --reverse_input_channels
+  - --transformations_config=$dl_dir/yolo-v3-tiny-tf/yolo-v3-tiny-tf.json
```
Comment (eaidova) [inline models/public/yolo-v3-tiny-tf/model.yml:34]: why mo extension file is not provided in the repo?

Comment (eaidova) [reply models/public/yolo-v3-tiny-tf/model.yml:34]: ok, i see it in archive, but I still want to know why not inside the repo

**Review by IRDonch — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

File: models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md [hunk 17]
```diff
@@ -0,0 +1,136 @@
...
+## Use Case and High-Level Description
+
+YOLO v3 Tiny is a real-time object detection model implemented with Keras\* from this [repository](https://github.com/david8862/keras-YOLOv3-model-set) and converted to TensorFlow\* framework. This model was pretrained on COCO\* dataset with 80 classes.
+
+## Conversion
+
+## Conversion
+
+1. Download or clone the original [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `d38c3d8` commit).
+2. Use the folllowing commands to get original model (named `yolov3_tiny` in repository) and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/blob/d38c3d865f7190ee9b19a30e91f2b750a31320c1/README.md)  file in the official repository):
```
Comment (IRDonch) [inline models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:12]:
```suggestion
2. Use the following commands to get original model (named `yolov3_tiny` in repository) and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/blob/d38c3d865f7190ee9b19a30e91f2b750a31320c1/README.md)  file in the official repository):
```

Comment (IRDonch) [reply models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md:12]: Okay, I don't feel like waiting for the demo CI job again. I'll just fix it later.
